### PR TITLE
docs(pylon.md): add bun requirement disclaimer

### PR DESCRIPTION
**Pull request by schettn**
Hey @yusukebe.

I want to add a disclaimer to figure out if the bun requirement of pylon is a dealbreaker for the community.

I am already working on a solution to support the same runtimes as Hono.

I just want to validate this feature. Could you share your personal opinion on this matter?

### CODE REVIEW

File: examples/pylon.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+:warning: **Pylon Requires Bun:** This project currently depends on the [Bun runtime](https://bun.sh). If you need support for a different runtime, please upvote and follow the discussion on Issue https://github.com/getcronit/pylon/issues/6 to help us prioritize it.
```
Comment (yusukebe) [inline examples/pylon.md:1]: Refer to this: https://vitepress.dev/guide/markdown#custom-containers

File: examples/pylon.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
...
+
+
```
Comment (schettn) [inline examples/pylon.md:3]:
```suggestion
::: warning
**Pylon Requires Bun:** This project currently depends on the [Bun runtime](https://bun.sh). If you need support for a different runtime, please upvote and follow the discussion on Issue https://github.com/getcronit/pylon/issues/6 to help us prioritize it.
:::

```

File: examples/pylon.md [hunk 3]
```diff
@@ -1,3 +1,7 @@
+::: warning
+**Pylon Requires Bun:** This project currently depends on the [Bun runtime](https://bun.sh). If you need support for a different runtime, please upvote and follow the discussion on Issue https://github.com/getcronit/pylon/issues/6 to help us prioritize it.
```
Comment (schettn) [inline examples/pylon.md:2]:
```suggestion
**Pylon Requires Bun:** This project currently depends on the [Bun runtime](https://bun.sh). If you need support for a different runtime, please upvote and follow the discussion on issue https://github.com/getcronit/pylon/issues/6 to help us prioritize it.
```

**Comment by yusukebe** [comment]
@schettn 

How about putting the warnings under the title?

CleanShot 2024-09-05 at 15 14 52@2x

**Comment by schettn** [comment]
> @schettn
> 
> How about putting the warnings under the title?
> 
> CleanShot 2024-09-05 at 15 14 52@2x

Yes makes sense when using the warning box.

File: examples/pylon.md [hunk 4]
```diff
@@ -1,3 +1,7 @@
+::: warning
+**Pylon Requires Bun:** This project currently depends on the [Bun runtime](https://bun.sh). If you need support for a different runtime, please upvote and follow the discussion on issue https://github.com/getcronit/pylon/issues/6 to help us prioritize it.
+:::
+
 # Pylon
```
Comment (schettn) [inline examples/pylon.md:6]:
```suggestion
# Pylon

::: warning
**Pylon Requires Bun:** This project currently depends on the [Bun runtime](https://bun.sh). If you need support for a different runtime, please upvote and follow the discussion on issue https://github.com/getcronit/pylon/issues/6 to help us prioritize it.
:::

```

**Review by yusukebe — APPROVED** [review]
LGTM!

**Comment by yusukebe** [comment]
@schettn Thanks!

**Comment by schettn** [comment]
@yusukebe 

Is it possible to enable some kind of preview deployment for the website?

**Comment by yusukebe** [comment]
@schettn 

Basically, I recommend previewing the document on your machine with running the dev server. But enabling preview deployment is not a bad idea. Can you create an issue on this repo?

**Comment by schettn** [comment]
@yusukebe 

Sure!

Yes I also preview it on my machine most oft the time. But when I make a commit through GitHub I cannot unless I install the GitHub CLI (which I do not want to)

**Pull request merged**